.readthedocs.yml

```diff
@@ -4,7 +4,7 @@ sphinx:
     configuration: doc/source/conf.py
 
 python:
-   version: "3.8"
+   version: "3.10"
    install:
```

Comment:
This should be updated to the improved way of defining the env on which to run rtd. We use that in https://github.com/arviz-devs/xarray-einstats/blob/main/.readthedocs.yaml